config/go.mod

```
@@ -1,3 +1,4 @@
+// Deprecated: use [go.opentelemetry.io/contrib/otelconf] instead.
```

```suggestion
// Deprecated: use go.opentelemetry.io/contrib/otelconf instead.
// This is the last release of this module.
```

I have not seen any tool which support `[]` for comments in `go.mod.